### PR TITLE
FIXED: Translations on the User Creation setup page

### PR DESCRIPTION
**Pull request by akemidx**
Translations have been added to the rest of the Create User page

- [x] Bug fix (non a bug, but needed)

# Checklist:

- [x] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [x] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by snipe** [comment]
This looks perfect - it's saying there are conflicts though?

**Comment by akemidx** [comment]
Oh, I think that the ending ]; was missing when i created the branch? maybe from my mistroking or typing? I am unsure, because it looked clean on my side when I submitted the PR. A glance at the 'resolve conflicts' makes it look liek it doesn't like that i moved that closing bracket? I don't have enough experience to really guess further.

File: resources/views/setup/user.blade.php [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-Create a User ::
+{{ trans('admin/user/general.create_user') }} ::
 @parent
 @stop
 
 {{-- Page content --}}
 @section('content')
 
-<p> This is the account information you'll use to access the site for the first time.  </p>
+<p>{{ trans('admin/user/general.create_user_page_explaination') }}</p>
```
Comment (snipe) [inline resources/views/setup/user.blade.php]: This is a small detail, but I think there's a typo in this key - "explaination" vs "explanation"

**Comment by snipe** [comment]
Ping! I'd love to get this into v6.1.0. <3

**Comment by akemidx** [comment]
should be all set

**Review by svpernova09 — APPROVED** [review]
Looks good from here.

**Pull request merged**